recipes/meraculous/meta.yaml

```diff
@@ -9,22 +9,23 @@ source:
 
 build:
     skip: True  # [osx]
-    number: 0
+    string: pl_5_22_{{PKG_BUILDNUM}}
```

Comment:
the build string looks like: `pl.5.22` see here for example: https://anaconda.org/bioconda/perl-bioperl/files